style.css

```diff
@@ -0,0 +1,211 @@
+
+html {
+  box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+body {
+  font-family: 'Museo Sans Cyrl 500';
+  font-style: normal;
+  font-size: 16px;
+  line-height: 19px;
+  color: #F8EEE4;
+  background: url(img/iswanto-arif-448003-unsplash.svg);
+  background-size: cover;
+
+}
+
+
+.wrapper {
+  margin: 0;
+}
+
+.header__wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: flex-end;
+  justify-content: space-between;
+  padding-top: 145px;
+  margin-bottom: 265px;
+}
+
+.header__logo {
+  display: flex;
+  flex-wrap: wrap;
+  margin-left: 247px;
+  
+}
+
+.header__list {
+  display: flex;
+  flex-wrap: wrap;
+  list-style: none;
+}
+
+.header__item {
+  margin-right: 34px;
+}
+
+.header__link {
+  font-family: 'Museo Sans Cyrl 500';
+  font-size: 22px;
+  line-height: 26px;
+  color: #F8EEE4;
+  text-decoration: none;
+}
+
+.header__link:hover,
+.header__link:focus,
+.header__link:active {
+  opacity: 75%;
+  text-decoration: underline;
+
+}
+
+.header__burger {
+  display: none;
+  width: 40px;
+  height: 28px;
+  position: relative;
+
+}
+
+.burger__line {
+  display: block;
+  width: 100%;
+  height: 3px;
+  background-color: #FCF5ED;
+  position: absolute;
+  left: 0;
+}
+
+.burger__line_first {
+  top: 0;
+}
+.burger__line_second {
+  top: 50%;
+  transform: translateY(-50%);
+}
+
+.burger__line_third {
+  bottom: 0;
+}
+
+
+.main__intro{
+display: flex;
+flex-wrap: wrap;
+justify-content: space-between;
+
+}
+
+
+
+
+  .intro__content {
+    
+    display: flex;
+    flex-wrap: wrap;
+    align-content: center;
+    position: relative;
+    margin-left: 247px;
+    
+
+  
+    
+  }
+
+
+.content__title {
+  font-family: 'PT Serif';
+  font-style: normal;
+  font-weight: bold;
+  font-size: 280px;
+  line-height: 362px;
+  color: #F8EEE4;
+  opacity: 0.4;
+  margin: 0;
+ 
+
+}
+
+.content__sibtitle {
+  position: absolute;
+  margin: 0;
+  font-family: 'PT Serif';
+  font-style: normal;
+  font-weight: bold;
+  font-size: 98px;
+  line-height: 127px;
+  color: #F8EEE4;
+  top: 60%;
+  left: 10%;
+
+}
+
+.content__text {
+  max-width: 550px;
+  font-size: 22px;
+  line-height: 26px;
+  margin-left: 268px;
+  margin-bottom: 78px;
+  
+}
+
+.content__button {
+  padding: 17px 87px;
+  font-size: 22px;
+  line-height: 26px;
+  color: #FFFF;
+  background: #D4BEA3;
```

Comment:
Codacy found an issue: [Expected "#D4BEA3" to be "#d4bea3" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/MyBali/pullRequest?prid=5781003)